### PR TITLE
Onboarding skribble-connector

### PR DESCRIPTION
**Pull request by phhung-axonivy**

### CODE REVIEW

File: skribble-connector-test/src_test/com/axonivy/connector/skribble/test/TestApiSubProcess.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  }
+
+  /**
+   * Testing SubProcesses, rather than full Business Processes is best practice: <br/>
+   * - it reduces the scope of your test: clear in/out and less infrastructure to setup <br/>
+   * - it drives a component driven architecture, and makes your processes therefore easier to re-use <br/>
+   */
+  @Test
+  void callSubProcess_getAllSignatureRequest(BpmClient bpmClient) {
+	  //var uri = Ivy.rest().client("Skribble").getUri();
```
Comment (phhung-axonivy) [inline skribble-connector-test/src_test/com/axonivy/connector/skribble/test/TestApiSubProcess.java:54]: Remove redundant code

File: skribble-connector-test/src_test/com/axonivy/connector/skribble/test/TestApiSubProcess.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	  //System.out.println("uri: "+ uri);
+	  
+	  var result = bpmClient.start()
+			.subProcess(Start.GET_ALL_SIGNATURE_REQUEST)
+			.execute();
+    
+    
+    SignatureRequestData data = result.data().last();
```
Comment (phhung-axonivy) [inline skribble-connector-test/src_test/com/axonivy/connector/skribble/test/TestApiSubProcess.java:62]: Use only 1 code style (spaces or tabs).
Could you also check other files?

File: skribble-connector/config/variables.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# == Variables ==
+# 
+# You can define here your project Variables.
+#
+Variables:
+  #set all paramaters for Skribble-connector
+  skribbleConnector:
+    #username
+    username: 'api_demo_axonivy_e989_0'
```
Comment (phhung-axonivy) [inline skribble-connector/config/variables.yaml:9]: It should be an empty value.

**Review by phhung-axonivy — COMMENTED** [review]
Hi @ivy-jh 
Could you check my comments and fix it?

File: skribble-connector-product/product.json [hunk 4]
```diff
@@ -50,8 +50,8 @@
       "data": {
         "dependencies": [
           {
-            "groupId": "MY-GROUP-ID",
-            "artifactId": "MY-PRODUCT-NAME",
+            "groupId": "com.axonivy.connector.skribble",
```
[the remark this answers is not in the document]
Comment (ivy-jh) [reply skribble-connector-product/product.json]: should work now, i removed the dropins dependency

**Comment by ivy-jh** [comment]
[phhung-axonivy](https://github.com/phhung-axonivy), 
Thanks for your help! i'm done with the your remarks and waiting for further reviews from you.

**Comment by phhung-axonivy** [comment]
Thank @ivy-jh 
Now, the code looks good. I will proceed to the next step.

**Comment by ivy-jh** [comment]
thanks [phhung-axonivy](https://github.com/phhung-axonivy) for merging! 👍 

i have just seen that the icon isnt displayed correctly on the market,  may you can have a look?
image

**Comment by phhung-axonivy** [comment]
Thanks for your feedback @ivy-jh 
We are investigating this problem.

**Comment by ivy-jh** [comment]
> Thanks for your feedback @ivy-jh We are investigating this problem.

what is the current status?

**Pull request merged**